src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -238,6 +243,30 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
     'press_releases_page.drupal.liquid',
     'press releases',
   );
+
+  // News Story listing page
+  const nsEntityUrl = createEntityUrlObj(drupalPagePath);
+  const nsObj = Object.assign(
```

Comment:
Do you need to use Object.assign here? I think you could just create a new object. You're not overriding updating a base object, so it doesn't seem like you're gaining anything from this.

Reply:
I was thinking the same thing!
I was following the pattern above. If i change it here, should i change it in the above ones as well?

Reply:
Yeah, I think that would make sense.